src/screens/CreateEventPage.js

```diff
@@ -8,151 +10,238 @@ class CreateEventPage extends Component {
 
       this.state = {
          fields: {},
-         errors: {}
+         errors: {},
+         formIsValid: true
+
       }
 
+      this.baseState = this.state
+
       this.handleValidation = this.handleValidation.bind(this)
    }
 
 
    handleValidation() {
```

Comment:
It would be better if we handled each inputs validation in a separate method rather than handling all of them in the same method. This will help us debug and make changes if necessary

Reply:
I tried to split these into sep validation methods, it actually made things much much worse, adds about 100 lines and makes it much harder to read